addons/website/controllers/main.py

```diff
@@ -1018,7 +1018,7 @@ def create_attachment(font, data):
     @http.route([
         '/website/action/<path_or_xml_id_or_id>',
         '/website/action/<path_or_xml_id_or_id>/<path:path>',
-    ], type='http', auth="public", website=True)
+    ], type='http', auth="public", website=True, readonly=True)
```

Comment:
That one does not look like a readonly one. We fetch an action then call its "run" method which can basically do anything. I'll remove it.